.github/workflows/zitadel-pr.yml

```diff
@@ -18,28 +18,34 @@ jobs:
         with:
           driver: docker
           install: true
-      - name: Test
-        run: docker build -f build/grpc/Dockerfile -t zitadel-base:local . && docker build -f build/zitadel/Dockerfile . -t zitadel-go-test --target go-codecov -o .artifacts/codecov
+      - name: Set up Docker Image Layer Caching
+        uses: satackey/action-docker-layer-caching@v0.0.11
```

Comment:
Lets remove this since we have no experience with it.
We can create a new PR down the road for this

Reply:
done by Elio